modules/channel-web/src/backend/api.ts

```diff
@@ -382,6 +382,9 @@ export default async (bp: typeof sdk, db: Database) => {
 
   router.post('/conversations/:userId/new', async (req: BPRequest, res: Response) => {
     const { userId, botId } = req.params
+    if (!userId) {
+      res.status(400).send({ message: 'Invalid user ID' })
```

Comment:
Missing a return, it will call both response

Reply:
Pretty sure it won't, request will be closed anyway. I did like you suggest.